common/tree/congestion_tree.go

```diff
@@ -1,18 +1,20 @@
 package tree
 
 import (
+	"encoding/json"
```

Comment:
@louisinger revert the changes to this file, we can't afford to store nodes as json strings in the db. Let's store a tree as a list of nodes with a foreign key referencing t the round, maybe we can use also another field to facilitate reconstructing the tree as [][]Node.

Reply:
Agreed, better to deconstruct and make the necessary SQL relations between data